Diagnostic/manifest.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version='1.0' encoding='utf-8' ?>
 <ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
-  <ProviderNameSpace>Microsoft.OSTCExtensions</ProviderNameSpace>
-  <Type>LinuxDiagnostic</Type>
-  <Version>2.3.9005</Version>
+  <ProviderNameSpace>Microsoft.Azure.Extensions</ProviderNameSpace>
+  <Type>TestLAD</Type>
+  <Version>2.3.9006.1</Version>
```

Comment:
It's not really needed at all, but it just shows that I publish the same extension with a different name, so my E2E tests can be done totally independent from this original name. If I use the original name and publish a test extension version, that test version is still made available to customer VMs, so I can't use the same name for a test extension version.